lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyCompletionParticipant.java

```diff
@@ -60,7 +60,7 @@ public void onAttributeValue(String valuePrefix, ICompletionRequest request, ICo
             variableName = valuePrefix.substring(valuePrefix.lastIndexOf("${"))
                     .replace("${", "");
             if (variableName.contains("}")) {
-                variableName = variableName.replace("}", "");
+                variableName = variableName.substring(variableName.lastIndexOf("}") + 1);
```

Comment:
if nothing appears after the last `}`, then we want to do completion on what was in the curly braces don't we?